util/omrutil/AtomicFunctions.cpp

```diff
@@ -60,3 +60,15 @@ issueWriteBarrier(void)
 {
 	VM_AtomicSupport::writeBarrier();
 }
+ 
+void
```

Comment:
Shouldn't both of these functions return a uintptr_t?  The corresponding C++ calls have a return and I suspect that users will typically want access to them.

Sorry for not catching that the first time.